libbeat/publisher/pipeline/module.go

```diff
@@ -95,7 +97,7 @@ func LoadWithSettings(
 
 	name := beatInfo.Name
 
-	queueBuilder, err := createQueueBuilder(config.Queue, monitors, settings.InputQueueSize)
+	queueBuilder, err := createQueueFactory(config.Queue, monitors, settings.InputQueueSize)
```

Comment:
```suggestion
	queueFactory, err := createQueueFactory(config.Queue, monitors, settings.InputQueueSize)
```

optional, very minor nit for readability.  since we are making a factory & New below takes a factory var name might be better as `queueFactory`

Reply:
ty, was trying to standardize on `Factory` since existing code mixed both, but missed this one!